### PR TITLE
Add docs for 0.13.2 rules integration package

### PR DESCRIPTION
**Pull request by brokensound77**
Add docs for rules released in the 0.13.2 rules integration package

[preview](https://security-docs_793.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-rules-downloadable-updates.html)

### CODE REVIEW

File: docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc
Comment (rw-access) [inline docs/detections/prebuilt-rules/prebuilt-rules-downloadable-updates.asciidoc:17]: 🤣 
```suggestion
activity as well as 4 new rules covering the recent
```

**Review by rw-access — APPROVED** [review]
LGTM when package ships.
how long is it typically between merging a PR and the elastic main docs updating?

**Comment by brokensound77** [comment]
Once this merges we will need to backport it to 7.13. Once that is merged, jobs run every ~30m to pickup and publish new changes.

I'll merge this and have the backport ready to go

**Pull request merged**
